src/main/test/java/ParserTest.java

```diff
@@ -0,0 +1,31 @@
+import Duke.Parser;
+import Duke.StringDatePair;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ParserTest {
+    @Test
+    public void inputEventTest(){
+        StringDatePair output = new Parser().parse(
+            "event Test Event /at 2020-10-28 1000",
+            Parser.commandType.INPUT_EVENT
+        );
+        assertEquals("Test Event ", output.getString());
+        assertEquals(LocalDateTime.of(2020, 10, 28, 10, 0)
+            , output.getDate());
+    }
+
+    @Test
+    public void inputDeadlineTest(){
+        StringDatePair output = new Parser().parse(
+            "deadline return book /by 2020-03-10 1800",
+            Parser.commandType.INPUT_DEADLINE
+        );
+        assertEquals("return book ", output.getString());
+        assertEquals(LocalDateTime.of(2020, 3, 10, 18, 0)
```

Comment:
Indentation for wrapped lines should be 8 spaces (i.e. twice the normal indentation of 4 spaces) more than the parent line.